tests/test_settings.py

```diff
@@ -2,11 +2,32 @@
 # Copyright (C) 2018-2022, earthobservations developers.
 # Distributed under the MIT License. See LICENSE for more info.
 import logging
+import re
 from concurrent.futures import ThreadPoolExecutor
 
 import numpy as np
 import pytest
 
+from wetterdienst import Settings
+
+
+def test_default_settings(default_settings, caplog):
+    caplog.set_level(logging.INFO)
+    default_settings = Settings.default()
```

Comment:
Should already be able to work without this line?
```suggestion
```

Reply:
Ah no, forgot to remove the fixture. If we use a fixture AND have the logging of the cache dir inside the settings then when using the fixture we cant catch the log because it has been cause in the fixture function scope but not in the default settings test itself.